tests/pr_test.go

```diff
@@ -1,60 +1,32 @@
-// Tests in this file are run in the PR pipeline and the continuous testing pipeline
 package test
 
 import (
+	"fmt"
+	"log"
+	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/gruntwork-io/terratest/modules/random"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
+const fullyConfigurableTerraformDir = "solutions/fully-configurable"
 
-// Ensure every example directory has a corresponding test
-const advancedExampleDir = "examples/advanced"
-const basicExampleDir = "examples/basic"
-
-func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:       t,
-		TerraformDir:  dir,
-		Prefix:        prefix,
-		ResourceGroup: resourceGroup,
-	})
-	return options
-}
-
-// Consistency test for the basic example
-func TestRunBasicExample(t *testing.T) {
+func TestTerraformCompleteTest(t *testing.T) {
 	t.Parallel()
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:      t,
+		TerraformDir: fullyConfigurableTerraformDir,
+	})
 
-	options := setupOptions(t, "mod-template-basic", basicExampleDir)
-
-	output, err := options.RunTestConsistency()
-	assert.Nil(t, err, "This should not have errored")
-	assert.NotNil(t, output, "Expected some output")
-}
-
-func TestRunAdvancedExample(t *testing.T) {
-	t.Parallel()
-
-	options := setupOptions(t, "mod-template-adv", advancedExampleDir)
-
-	output, err := options.RunTestConsistency()
-	assert.Nil(t, err, "This should not have errored")
-	assert.NotNil(t, output, "Expected some output")
-}
-
-// Upgrade test (using advanced example)
-func TestRunUpgradeExample(t *testing.T) {
-	t.Parallel()
+	prefix := fmt.Sprintf("ocp-ai-da-%s", strings.ToLower(random.UniqueId()))
```

Comment:
you don't need to do this, the `TestOptionsDefault` supports generating the prefix for you:

```
options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
		Testing:      t,
		TerraformDir: fullyConfigurableTerraformDir,
                Prefix: "ocp-ai-da"
	})

options.TerraformVars = map[string]interface{}{
		"prefix": options.Prefix,
	}
```